<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImportVICEDialog</class>
 <widget class="QDialog" name="ImportVICEDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>339</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import VICE Snapshot</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_2">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Filename</string>
        </property>
        <property name="buddy">
         <cstring>lineEdit</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="1">
       <widget class="ImportVICEScreenRAMWidget" name="widgetScreenRAM" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="ImportVICECharsetWidget" name="widgetCharset" native="true"/>
      </item>
      <item row="2" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Charset address</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxCharset">
          <property name="prefix">
           <string>$</string>
          </property>
          <property name="maximum">
           <number>63488</number>
          </property>
          <property name="singleStep">
           <number>2048</number>
          </property>
          <property name="displayIntegerBase">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="1">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Screen RAM address</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxScreenRAM">
          <property name="prefix">
           <string>$</string>
          </property>
          <property name="maximum">
           <number>64512</number>
          </property>
          <property name="singleStep">
           <number>1024</number>
          </property>
          <property name="displayIntegerBase">
           <number>16</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Charset:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Screen RAM (map):</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxMulticolor">
     <property name="text">
      <string>Multicolor</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxGuessColors">
     <property name="text">
      <string>Guess Charset Colors</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxDisplayGrid">
     <property name="text">
      <string>Display Grid</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkBoxInvalidAddresses">
     <property name="text">
      <string>Allow Invalid VIC addresses</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6"/>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_import">
       <property name="text">
        <string>Import</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImportVICECharsetWidget</class>
   <extends>QWidget</extends>
   <header>importvicecharsetwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ImportVICEScreenRAMWidget</class>
   <extends>QWidget</extends>
   <header>importvicescreenramwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEdit</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>spinBoxCharset</tabstop>
  <tabstop>spinBoxScreenRAM</tabstop>
  <tabstop>checkBoxMulticolor</tabstop>
  <tabstop>checkBoxGuessColors</tabstop>
  <tabstop>checkBoxDisplayGrid</tabstop>
  <tabstop>checkBoxInvalidAddresses</tabstop>
  <tabstop>pushButton_cancel</tabstop>
  <tabstop>pushButton_import</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
